### ISSUE TITLE
Can i takeover subdomain with cname that refers cdn.cloudflare.com

### ISSUE THREAD

**Issue by yousefmoh15** [EdOverflow/can-i-take-over-xyz#193]
I have found a subdomain like sub.example.com which alias to sub.example.com.cdn.cloudflare.com
 when I request that subdomain it gives me
 ( 404 not found) 
so can i take over it or not ?

**Comment by Beastvour** [comment]
Any Update ? ,. Did you take over ? ,. 
This site can’t provide a secure connection
sub.example.eu uses an unsupported protocol.
ERR_SSL_VERSION_OR_CIPHER_MISMATCH

**Comment by isira-adithya** [comment]
I have the same question? Anyone knowss how to do this?

**Comment by indianajson** [comment]
Subdomains pointing to `sub.example.com.cdn.cloudflare.com` utilize a CNAME Zone on Cloudflare which requires a Business or Enterprise account and requires TXT record verification at the authoritative DNS per this article titled [Understanding a CNAME Setup](https://support.cloudflare.com/hc/en-us/articles/360020348832-Understanding-a-CNAME-Setup). Thus, **takeovers are impossible.**

**Comment by isira-adithya** [comment]
Oh thanks a lot for the clarification. 👌

**Comment by nhl0010** [comment]
thanks

**Comment by abusabiha** [comment]
In Some Place, we see "404 Not Found. nginx".

when we dig this, we saw some ip address. but not found is it from where.

Can anybody tell me how can we find it and is it vulnerable or not?

**Comment by stoick-the-vast** [comment]
I don't think so we can takeover subdomain that point towards the IP. Means cname like 18.6.23.54 not possible to takeover
